docs/build/tutorials/platform/subnets/create-a-evm-blockchain-on-subnet-with-avalanchejs.md

```diff
@@ -0,0 +1,545 @@
+
+
+## Requirements
+
+* [Go](https://golang.org/doc/install) version >= 1.16.8
+* [NodeJS](https://nodejs.org/en) Node >= 10.16 and npm >= 5.6
+
+## Project structure
+
+Open the terminal and make a new directory at your desired location. We will keep all the binaries and other codes in this folder.
+
+```bash
+mkdir subnet-evm-demo
+cd subnet-evm-demo
+```
+
+## Setting up AvalancheGo and Subnet EVM Binaries
+
+Clients interact with Avalanche blockchain by issuing API calls to the nodes running `AvalancheGo`. We have to clone the repository, build its binary, and run it. If we want to deploy our blockchain, we have to put the blockchain's VM binary inside the `build/plugins` directory. Here we will also clone the `subnet-evm` repository, build the VM's binary, and copy it to the AvalancheGo's `build/plugins` directory. Follow the steps below -
+
+### Clone AvalancheGo Repository
+
+```bash
+git clone https://github.com/ava-labs/avalanchego
+cd avalanchego
+```
+
+### Build Binary
+
+Running the below command will create an `avalanchego` binary inside the `build/` directory and will install the coreth `evm` binary inside the `build/plugins` directory.
+
+```bash
+./scripts/build.sh
+```
+
+### Clone Subnet-EVM Repository
+
+Move to the `subnet-evm-demo` directory, and clone the repo.
+
+```bash
+git clone https://github.com/ava-labs/subnet-evm
+cd subnet-evm
+```
+
+### Build Binary and Copy it to AvalancheGo Plugins
+
+Now run the following command to build the VM's binary inside the `build/` directory, named as `srEXiWaHuhNyGwPUi444Tu47ZEDwxTWrbQiuD7FmgSAQ6X7Dy`. It is the id for this VM and corresponds to the string "**subnetevm**" zero-extended in a 32-byte array and encoded in CB58. Then copy it to AvalancheGo's `build/plugins` directory.
+
+```bash
+./scripts/build.sh build/srEXiWaHuhNyGwPUi444Tu47ZEDwxTWrbQiuD7FmgSAQ6X7Dy
+
+cp build/srEXiWaHuhNyGwPUi444Tu47ZEDwxTWrbQiuD7FmgSAQ6X7Dy ../avalanchego/build/plugins
+```
+
+You can directly build the `subnet-evm` binary inside the plugins folder as well, by passing its location as the argument.
+
+## Setting up Local Avalanche Network
+
+For the development purpose, we can use **Avalanche Network Runner (ANR)**. It helps us in simulating the actual network. For this tutorial, we will be installing ANR binary and will interact with the network through RPCs.
```

Comment:
Add a link to https://docs.avax.network/build/tools/network-runner/ for **Avalanche Network Runner (ANR)**